### PR TITLE
Added Stable files for SelfHelp

### PR DESCRIPTION
**Pull request by siddiavinashmsft**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/selfhelp/Azure.ResourceManager.SelfHelp/src/Generated/SelfHelpDiagnosticData.cs [hunk 1]
```diff
@@ -52,10 +52,10 @@ public SelfHelpDiagnosticResourceData()
         /// Array of Diagnostics.
         /// Serialized Name: DiagnosticResource.properties.diagnostics
         /// </param>
-        internal SelfHelpDiagnosticResourceData(ResourceIdentifier id, string name, ResourceType resourceType, SystemData systemData, IDictionary<string, string> globalParameters, IList<DiagnosticInvocation> insights, string acceptedTime, ProvisioningState? provisioningState, IReadOnlyList<SelfHelpDiagnostic> diagnostics) : base(id, name, resourceType, systemData)
```
Comment (archerzz) [inline sdk/selfhelp/Azure.ResourceManager.SelfHelp/src/Generated/SelfHelpDiagnosticData.cs:55]: Is `acceptedTime` really a string? Could it be a `DateTimeOffset`? See an example [here](https://github.com/Azure/azure-sdk-for-net/blob/e4fce1607d65d709b19c81cc78969cd08b804244/sdk/hdinsight/Azure.ResourceManager.HDInsight/src/autorest.md?plain=1#L146).

**Review by archerzz — COMMENTED** [review]
@siddiavinashmsft I left one comment. Others look good to me. Thanks.

**Review by archerzz — COMMENTED** [review]
@siddiavinashmsft I left one comment. Others look good to me. Thanks.

File: sdk/selfhelp/Azure.ResourceManager.SelfHelp/src/Generated/SelfHelpDiagnosticData.cs (hunk 1 above)
Comment (siddiavinashmsft) [reply sdk/selfhelp/Azure.ResourceManager.SelfHelp/src/Generated/SelfHelpDiagnosticData.cs:55]: This was a decision taken a while back to keep it string. We will be fixing it in the future versions.

**Review by archerzz — APPROVED** [review]

File: sdk/selfhelp/Azure.ResourceManager.SelfHelp/api/Azure.ResourceManager.SelfHelp.netstandard2.0.cs [hunk 2]
```diff
@@ -1,167 +1,167 @@
...
+        public static Azure.ResourceManager.SelfHelp.Models.SelfHelpError SelfHelpError(string errorCode = null, string errorType = null, string errorMessage = null, System.Collections.Generic.IEnumerable<Azure.ResourceManager.SelfHelp.Models.SelfHelpError> errorDetails = null) { throw null; }
+        public static Azure.ResourceManager.SelfHelp.Models.SelfHelpSolutionMetadata SelfHelpSolutionMetadata(Azure.Core.ResourceIdentifier id = null, string name = null, Azure.Core.ResourceType resourceType = default(Azure.Core.ResourceType), Azure.ResourceManager.Models.SystemData systemData = null, string solutionId = null, string solutionType = null, string solutionDescription = null, System.Collections.Generic.IEnumerable<System.Collections.Generic.IList<string>> requiredParameterSets = null) { throw null; }
     }
-    public partial class CheckNameAvailabilityContent
+    public partial class SelfHelpCheckNameAvailabilityContent
```
Comment (ArthurMa1978) [inline sdk/selfhelp/Azure.ResourceManager.SelfHelp/api/Azure.ResourceManager.SelfHelp.netstandard2.0.cs:57]: Rename to `SelfHelpNameAvailabilityContent`

File: sdk/selfhelp/Azure.ResourceManager.SelfHelp/api/Azure.ResourceManager.SelfHelp.netstandard2.0.cs [hunk 3]
```diff
@@ -1,167 +1,167 @@
...
     {
-        public CheckNameAvailabilityContent() { }
-        public string Name { get { throw null; } set { } }
+        public SelfHelpCheckNameAvailabilityContent() { }
+        public string ResourceName { get { throw null; } set { } }
         public string ResourceType { get { throw null; } set { } }
     }
-    public partial class CheckNameAvailabilityResponse
+    public partial class SelfHelpCheckNameAvailabilityResult
```
Comment (ArthurMa1978) [inline sdk/selfhelp/Azure.ResourceManager.SelfHelp/api/Azure.ResourceManager.SelfHelp.netstandard2.0.cs:63]: Rename to `SelfHelpNameAvailabilityContent`

**Review by ArthurMa1978 — APPROVED** [review]

**Review by ArthurMa1978 — APPROVED** [review]

File: sdk/selfhelp/Azure.ResourceManager.SelfHelp/CHANGELOG.md [hunk 4]
```diff
@@ -1,12 +1,24 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0 (2022-06-02)
```
Comment (ArthurMa1978) [inline sdk/selfhelp/Azure.ResourceManager.SelfHelp/CHANGELOG.md:3]: What is the expected released date ?

**Pull request merged**